src/services/revocation.js

```diff
@@ -32,60 +32,183 @@ export class RevocationStatusServiceImpl {
    * Checks revocation status of UCAN delegations via Storage UCAN Service
    *
    * @param {Ucanto.Proof[]} proofs - Array of UCAN proofs to check
+   * @param {string} spaceDID - Space DID to validate delegation context
    * @param {import('../types/env.js').Env} env - Environment configuration
    * @returns {Promise<import('@ucanto/server').Result<boolean, import('@ucanto/server').Failure>>}
    */
-  async checkStatus (proofs, env) {
-    const safeProofs = proofs || []
-
+  async checkStatus (proofs, spaceDID, env) {
+    const delegations = (proofs || []).map(p => /** @type {import('@ucanto/interface').Delegation} */ (p))
+    if (delegations.length === 0) {
+      this.auditLog.logSecurityEvent('revocation_check_success', {
```

Comment:
This audit log is tightly coupled to this client library, but it feels like it's something that is probably used in the application for other things. Why not just return the information you want to be logged in the result and then log it in the application?